.github/actions/setup/action.yml

```diff
@@ -20,23 +20,18 @@ runs:
       with:
         go-version: ${{ inputs.go_version }}
 
+    - name: Install pnpm
+      uses: pnpm/action-setup@v4.0.0
```

Comment:
This action is already defined at lines 33-37, so I don't understand this addition?

Reply:
You need to install pnpm first then the setup node task allows you to use a cache, otherwise this task fails if pnpm isn't installed. Then you run the pnpm task next to actually install the packages.  The GitHub action has this documented like that as well.